astroquery/solarsystem/neocc/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+astroquery.solarsystem.neocc
+--------------------------
+
+a collection of data services provided by NEOCC
+"""
+
+# from ...esa.neocc import neocc, ESAneoccClass
```

Comment:
do use absolute imports, and remove the comment. Also add an ``__all__``

Reply:
Done